docs/swagger/spec.yml

```diff
@@ -20,8 +20,12 @@ consumes:
 tags:
```

Comment:
I can see utility in allowing users to filter their bucket scenes: eg. they want to remove all scenes with a cloud coverage > 50 %. Would require some UI redesign though, to fit filters in the library view.